amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ProcessTask.java

```diff
@@ -45,6 +45,7 @@
 @KinesisClientInternalApi
 public class ProcessTask implements ConsumerTask {
     private static final String PROCESS_TASK_OPERATION = "ProcessTask";
+    private static final String APPLICATION_LEVEL_METRICS = "ApplicationLevelMetrics";
```

Comment:
Naming: doesn't say explicitly what this metric is about. 
Suggestion: AppMillisBehindLatest / something else that speaks about the metric and the visibility level

Reply:
I read the public documentation and found "ApplicationLevelMetrics" not descriptive enough. This new metric we are adding belongs to https://docs.aws.amazon.com/streams/latest/dev/monitoring-with-kcl.html#kcl-metrics-per-app. Refer to this naming convention on Operation field, and get feedback from PM on the naming.

Reply:
I showed a demo to the PM previously, but yeah I agree, will talk with the PM to confirm the names.

Reply:
After talking with Nihar, we decided to use `ApplicationTracker` as the operation, and the code is updated now.